docs/examples/pydata.md

```diff
@@ -3,6 +3,8 @@ file_format: mystnb
 kernelspec:
   name: python3
   display_name: Python 3
+mystnb:
+  execution_mode: cache
```

Comment:
this should prevent us from re-running this page every single time we build the docs